cves/CVE-2013-0844.yml

```diff
@@ -344,7 +357,10 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer:
+  answer: |
+    Improved unit testing and code coverage could have gone a long way to catch vulnerabilities like this one. Additionally
+    and perhaps more important for this development team would be improved code reviews. The mistake was simple yet very dangerous. Another
```

Comment:
```suggestion
    and perhaps more important for this, the development team would be improved code reviews. The mistake was simple yet very dangerous. Another
```

Reply:
I have updated the grammar, the meaning should be "perhaps more important for this development team,"